src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1877,6 +1906,25 @@ subroutine gsi_fv3ncdf2d_read(fv3filenamegin,it,ges_z,ges_t2m,ges_q2m)
           iret=nf90_inquire_dimension(gfile_loc,k,name,len)
           dim(k)=len
        enddo
+
+!---   check the existence of significant wave height (howv) in 2D FV3-LAM firstguess file
+!      if howv is set in anavinfo (as i_howv_3dda=1), then check its existence in firstguess,
+!      but if it is not found in firstguess, then stop GSI run and set i_howv_3dda = 0.
+       if ( i_howv_3dda == 1 ) then
+         iret = nf90_inq_varid(gfile_loc,'howv',id_howv)
+         if ( iret /= nf90_noerr ) then
+           iret = nf90_inq_varid(gfile_loc,'HOWV',id_howv) ! double check with name in uppercase
+         end if
+         if ( iret /= nf90_noerr ) then
+           i_howv_3dda = 0                ! howv does not exist in firstguess, then stop GSI run.
```

Comment:
Line 1919 is not needed as GSI dies immediately.

Reply:
@guoqing-noaa 
I agree, since GSI is going to abort immediately in this case.
My thought about keeping this line (i_howv_3dda = 0) here, is to mark this event (howv is NOT in firstguess) and can be used later, if in the future some users, eg. NCO, would not like to stop the running of GSI just just due to missing of howv, setting i_howv_3dda = 0 will be useful. And keeping this line here does no harm to the efficiency and accuracy of code.
Thank you!